test/extended/util/adminack.go

```diff
@@ -36,6 +37,24 @@ var adminAckGateRegexp = regexp.MustCompile(adminAckGateFmt)
 // admin-acks configmap to ack the given admin-ack gate. Once all gates have been ack'ed, the test waits for the
 // Upgradeable condition to change to true.
```

Comment:
Replace with:
`// If the gate has been ack'ed the test verifies that the Upgradeable condition does not complain about the ack. Test`
`// then clears the ack and verifies that the Upgradeable condition complains about the ack. Test then sets the ack`
`// and verifies that the Upgradeable condition no longer complains about the ack.`